internal/app/machined/internal/phase/network/network.go

```diff
@@ -56,6 +56,12 @@ func (task *InitialNetworkSetup) runtime(r runtime.Runtime) (err error) {
 		return err
 	}
 
+	log.Println("overlaying config network configuration")
```

Comment:
nit: Can we say something like `"overriding discovered network configuration with use defined options"`? I get `overlay`, but something like the above is super explicit, which I find to be helpful in the early boot.